tests/foreman/maintain/test_upgrade.py

```diff
@@ -83,6 +95,69 @@ def test_positive_repositories_validate(sat_maintain):
     assert skip_message in result.stdout
 
 
+@pytest.mark.no_containers
+@pytest.mark.parametrize(
+    'custom_host',
+    [
+        {'deploy_rhel_version': '8', 'deploy_flavor': 'satqe-ssd.disk.xxxl'},
+        {'deploy_rhel_version': '8', 'deploy_flavor': 'satqe-ssd.standard.std'},
+    ],
+    ids=['default', 'medium'],
+    indirect=True,
+)
+def test_negative_pre_upgrade_tuning_profile_check(request, custom_host):
+    """Negative test that verifies a satellite with less than
+    tuning profile hardware requirements fails on pre-upgrade check.
+
+    :id: 240bb97e-7353-4397-8d0e-228c3593c8cc
+
+    :steps:
+        1. Check out a RHEL instance with less than tuning requirements
+        2. Install Satellite
+        3. Run pre-upgrade check
+
+    :expectedresults: Pre-upgrade check fails.
+    """
+    profile = request.node.callspec.id
+    sat_version = ".".join(settings.server.version.release.split('.')[0:2])
+    # Register to CDN for RHEL8 repos, download and enable last y stream's ohsnap repos,
+    # and enable the satellite module and install it on the host
+    custom_host.register_to_cdn()
+    last_y_stream = last_y_stream_version(settings.server.version.release)
+    custom_host.download_repofile(product='satellite', release=last_y_stream)
+    custom_host.execute('dnf -y module enable satellite:el8 && dnf -y install satellite')
+    # If testing <= 6.13 we can run installer with --disable-system-checks,
+    # otherwise we need to set the --tuning to development and then change the tuning
+    # profile manually to get around the installer check
+    if float(sat_version) <= 6.13:
```

Comment:
we branch so you don't have to put this if clause

to master put else clause (+ cherrypick to 6.13.z)
when cherrypicking to 6.12.z amend and use part from if clause

Reply:
You're right I should base this off the branch. However, the if part would be for current release (6.13) and then when we branch 6.14 I'll have to switch it to the else clause. Maybe we should merge with just the if statement part and then when this fails on 6.14 I can switch to use the else part.